valenstical/ireporter
# #162286888 Chore: Setup unit test for routes

**What does this PR do?**
This PR implements unit test for the public route ('/') and the main API entry route ('/api/v1')

**Description of tasks**

* Write tests for the public route / and ensure that it passes
* Write tests for the main entry route the api /api/v1 and ensure that it passes
* Write tests for invalid routes /INVALID_ROUTE and ensure that it passes

**How can this PR be tested**
* Using POSTMAN, set the method to GET
* Make a request to https://ireporter-nigeria.herokuapp.com/ or https://ireporter-nigeria.herokuapp.com/api/v1
* Request should return a 200 status code with success message or 404 status code with an error message for valid and invalid routes respectively

**What are the relevant pivotal tracker stories?**
[#162286888](https://www.pivotaltracker.com/story/show/162286888)

**Screenshot**
N/A
